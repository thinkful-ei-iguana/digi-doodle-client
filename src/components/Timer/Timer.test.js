import React from 'react'
import ReactDOM from 'react-dom'
import Timer from './Timer'
import renderer from 'react-test-renderer'
import { BrowserRouter } from 'react-router-dom'

describe(`Timer component`, () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');

        ReactDOM.render(
            <BrowserRouter>
                <Timer />
            </BrowserRouter>, div);

        ReactDOM.unmountComponentAtNode(div)
    })

    it('renders the appropriate time', () => {
        const state = {
            time: 0
        }

        const timer = renderer.create(<Timer {...state} />).toJSON();
        expect(timer).toMatchSnapshot();
    })

})