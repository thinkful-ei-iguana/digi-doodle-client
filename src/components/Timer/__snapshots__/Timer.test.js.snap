// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer component renders the appropriate time 1`] = `
<div
  className="timer-container"
/>
`;
